doc/news/changes.h

```diff
@@ -134,6 +134,12 @@ inconvenience this causes.
 
 
 <ol>
+  <li> Improved: Computation of gradients in FE_PolyTensor and its subclasses
+  also takes into account the term corresponding to the mapping gradient.
```

Comment:
You're being too modest here. How about
```
  Fixed: The computation of gradients in FE_PolyTensor and its derived classes (in particular in
  FE_RaviartThomas and FE_Nedelec) forgot to account for terms that appear on non-affine
  cells. Consequently, the computed gradients did not match the actual derivatives of the values
  these elements report. This is now fixed.
```
Feel free to put this into the "General" section of the file.